linux/main.cpp

```diff
@@ -494,7 +515,7 @@ public slots:
         process.waitForFinished();
         QString output = process.readAllStandardOutput().trimmed();
         LOG_DEBUG("Default sink: " << output);
-        return output.contains("bluez_card." + connectedDeviceMacAddress);
+        return (output.contains("bluez_card.") || output.contains("bluez_output.")) && output.contains(connectedDeviceMacAddress);
```

Comment:
On my device the name of the output was different, with this modifications, both versions should work